data/src/main/java/cmm/apps/esmorga/data/event/EventRepositoryImpl.kt

```diff
@@ -9,34 +9,21 @@ import cmm.apps.esmorga.data.user.datasource.UserDatasource
 import cmm.apps.esmorga.data.user.model.UserDataModel
 import cmm.apps.esmorga.domain.event.model.Event
 import cmm.apps.esmorga.domain.event.repository.EventRepository
-import cmm.apps.esmorga.domain.result.ErrorCodes
-import cmm.apps.esmorga.domain.result.EsmorgaException
-import cmm.apps.esmorga.domain.result.Success
 
 class EventRepositoryImpl(private val localUserDs: UserDatasource, private val localEventDs: EventDatasource, private val remoteEventDs: EventDatasource) : EventRepository {
 
-    override suspend fun getEvents(forceRefresh: Boolean): Success<List<Event>> {
+    override suspend fun getEvents(forceRefresh: Boolean, forceLocal: Boolean): List<Event> {
```

Comment:
why is the forceLocal necessary? 🤔

Reply:
You can see it in GetEventListUseCase. When remote call has thrown an exception for no connection, it goes back to the repository to retrieve the data only in cache. It's the way we thought to have repositories return simple types and still being able to return error and cached data.